eLife.js

```diff
@@ -84,7 +84,7 @@ function doWeb(doc, url) {
 
 
 function scrape(doc, url) {
-
+	url = url.split('?')[0] // Remove URL parameters like ?utm_source
 	var risURL = url.replace(/#.+/, "") + ".ris";
```

Comment:
why not just add this here instead, so `url.replace(/([#?].+/, "") + ".ris";`?

Reply:
Ah, sure why not. I only had a cursory look and thought that the url was also used to construct the PDF url, but that isn't the case.